Branch2018/test1/Program.cs

```diff
@@ -0,0 +1,338 @@
+                                        case 1: SalesReport(salesList);
+                                                break; 
+                                                
+                                        case 2: CustomersReport(customersList);
+                                                break;
+                                                
+                                        case 3: CustomerReportAge(customersList);
+                                                break; 
+                                                
+                                        case 4: ProductsReport(productsList);
+                                                break; 
+                                                
+                                        case 5: break;
+                                    }
+                                }
+                            }
+                            break;
+                }
+            }
+            Console.WriteLine("Thank for your visit.");
+        }
+
+        public static void SalesReport(List<Sale> salesList)
+        {
+            bool p = true;
+            Console.WriteLine("Total day sales :"+ salesList.Count);
+            if(salesList.Count == 0)
+            {
+                Console.WriteLine("There are not registered sales.");
+                p = false; 
+            }
+            else
+                {
+                    foreach (Sale sale in salesList)
+                    {
+                            if (p)
+                            {                            
+                                Console.WriteLine("--------------");
+                                Console.WriteLine("Customer code :" + $"{sale.CustomerId},  Product code: {sale.ProductId}, Sales description : {sale.Description}");                               
+                            }
+                    }
+                }
+        }
+
+        public static void CustomersReport(List<Customer> customersList)
+        {
+            bool p = true;
+            Console.WriteLine("Customers older to 55 years:");
+
+            if(customersList.Count == 0)
+            {
+                Console.WriteLine("The are not registered customers");
+            }
+            else
+                {
+                    foreach (Customer customer in customersList)
+                    {
+                            if (customer.Age > 55)
+                            {
+                                p = false;
+                                Console.WriteLine("--------------");
+                                Console.WriteLine("---------Customer full name :" + $"{customer.FullName  } {customer.LastName}, Age: {customer.Age}, Birhtday Month: {customer.MonthBirthday}");
+                            }
+                            if(p)
+                                Console.WriteLine("There are not customers older to 55 yeards old");                

```

Comment:
even if is not necessary use brackets when it's only one line of code, the coding standards says that it's better because the code could change and next time is possible to add more code and there could be throw an exception if next developer forget to put brackets to all code.